api/client/run.go

```diff
@@ -199,8 +224,9 @@ func (cli *DockerCli) CmdRun(args ...string) error {
 	}()
 
 	//start the container
-	if _, _, err = readBody(cli.call("POST", "/containers/"+createResponse.ID+"/start", nil, nil)); err != nil {
-		return err
+	if _, _, err := readBody(cli.call("POST", "/containers/"+createResponse.ID+"/start", nil, nil)); err != nil {
+		cmd.ReportError(err.Error(), true)
```

Comment:
I'm thinking the 2nd param should be `false` here.
```
$ docker run -ti ubuntu /tmp
exec: "/tmp": permission denied
docker: Error response from daemon: Contained command could not be invoked..
See '/root/docker/bundles/1.9.0-dev/binary/docker run --help'.
```
I think it makes sense to show the short help message in cases where we think there might be a docker specific error on the cmd line (like a bad option), but in cases where we've made it all the way to the container.start() we're probably into the user (or cmd) space and so I'm not sure showing the help message helps.  Keeping it on the parsing error and create error, above, are ok though.